pkg/services/alerting/notifiers/pagerduty_test.go

```diff
@@ -25,7 +25,28 @@ func TestPagerdutyNotifier(t *testing.T) {
 				So(err, ShouldNotBeNil)
 			})
 
-			Convey("auto resolve should default to false", func() {
+			Convey("severity should override default", func() {
+				json := `{ "integrationKey": "abcdefgh0123456789", "severity": "info" }`
+
+				settingsJSON, _ := simplejson.NewJson([]byte(json))
```

Comment:
Could you please verify that no error is returned here?

Reply:
I'll add it for all, since this is the same pattern as the other test cases.

Reply:
Done.